phpseclib/Net/SSH2.php

```diff
@@ -4637,7 +4637,7 @@ public static function getSupportedEncryptionAlgorithms()
             'twofish192-cbc', // OPTIONAL          Twofish with a 192-bit key
             'twofish256-cbc',
             'twofish-cbc',    // OPTIONAL          alias for "twofish256-cbc"
-                              //                   (this is being retained for historical reasons)
```

Comment:
Another place I think we should do `// @codingStandardsIgnoreStart` and `// @codingStandardsIgnoreEnd`

Reply:
Not needed, I can manually align